easybuild/easyconfigs/v/vitessceR/vitessceR-0.99.0-20230110-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'Bundle'
+
+name = 'vitessceR'
+# the github repo has no releases / tags, that's why we have to use commit...
+version = '0.99.0-20230110'
+versionsuffix = '-R-%(rver)s'
+local_commit = '8c67168'
+
+homepage = 'https://github.com/vitessce/vitessceR'
+description = "Vitessce is a visual integration tool for exploration of spatial single-cell experiments."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('R', '4.2.1'),
+    ('R-bundle-Bioconductor', '3.15', versionsuffix),
+    ('libsodium', '1.0.18'),
```

Comment:
This should probably be moved to R-4.2.1 as well, as the sodium extension is now causing the builds to fail

Reply:
Hmm, maybe that's a reason to keep the `sodium` extension here...

Reply:
That would imply also moving `plumber` back here, since that requires `sodium`

Reply:
Yeah, that could make sense. Prevents the dep-list for R growing too much... I agree, this would also be my preference, to keep `libsodium`, `sodium` and anything that is required by `sodium` (I guess that's `plubmer` then?) in `vitessceR`.

Reply:
It's `plubmer` that requires `sodium`, not the other way around.

But yes, keeping `libsodium` dep + `sodium` extension in `vitesseR` makes most sense.

@deniskristak Can you change that?

Reply:
done